### PR TITLE
clearification of the advanced network setup in section "Host KVM installation"

### PR DESCRIPTION
**Pull request by Hudratronium**
I rewrote the setup-description, thinking is gives a better overview of the logical topology of the setup.

### CODE REVIEW

File: source/installguide/hypervisor/kvm.rst [hunk 1]
```diff
@@ -694,7 +693,7 @@ Make sure it looks similar to:
    TYPE=Ethernet
    BRIDGE=cloudbr0
 
-We now have to configure the VLAN interfaces:
+We now have to configure the second network-interface:
```
Comment (DaanHoogland) [inline source/installguide/hypervisor/kvm.rst:696]:
```suggestion
We now have to configure the second network-interface for use in guest VLANs:
```

**Review by DaanHoogland — COMMENTED** [review]
looks ok, minor changes

**Review by DaanHoogland — APPROVED** [review]

**Comment by DaanHoogland** [comment]
requesting docbuild

File: source/installguide/hypervisor/kvm.rst [hunk 2]
```diff
@@ -645,11 +645,10 @@ although a reboot is recommended to see if everything works properly.
...
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In the Advanced networking mode is most common to have (at least) two physical interfaces.
-In this example we will again have the hypervisor management interface on cloudbr0 on the 
-untagged (native) VLAN. But now we will have a bridge on top of our additional interface (eth1)
-for public and guest traffic with no VLANs applied by us - CloudStack will add the VLANs
-as required.
+In the Advanced networking mode is most common to have (at least) two physical interfaces per hypervior-host.
+We will use the interface eth0 linked to the bridge 'cloudbr0' using the untagged (native) VLAN for hypervisor management.
+Additionally we configure a the second interface for usage with the bridge 'cloudbr1' for public and guest traffic.
```
Comment (GabrielBrascher) [inline source/installguide/hypervisor/kvm.rst:650]: `Additionally we configure a the second interface`
:arrow_down: I would suggest removing `a`, but you might remove `the` instead, if you prefer
`Additionally, we configure the second interface`

**Review by GabrielBrascher — COMMENTED** [review]
Thanks for the PR @Hudratronium. I have added a small suggestion.

File: source/installguide/hypervisor/kvm.rst (hunk 2 above)
Comment (Hudratronium) [reply source/installguide/hypervisor/kvm.rst]: Added suggestions  - thanks for checking and reading carefully :-)

**Pull request merged**